multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -51,6 +62,7 @@ def is_categorical_column(
     else:
         if threshold is None:
             if is_label:
+                # TODO(?) The following logic will be problematic if the task is few-shot learning.
```

Comment:
@Linuxdex @Raldir Will be related to few-shot learning. We should fix the logic in a future PR.